content/_index.md

```diff
@@ -11,9 +11,9 @@ Welcome to the SUSE CAP Sandbox!
 
 If you would like to skip the introduction and just get logged in and start playing around, [click here](/quickstart/)
   
-# Key Developer Benefits to Platform as a Service
+## Key Developer Benefits to Platform as a Service
 
-Imagine you just got a new laptop. First thing you do is slab your favorite OS on it and then the fun begins. You have to install your favorite editor or IDE, compilers, libraries, debug tools and what have you. Fast forward a few hours and you're slowly getting to a point where your new tool becomes usable. Couple of weeks later something in your tool-chain breaks and you have to spend time to fix it. Updates need to be made, dependencies to be considered and managed and so on. Bottom line is, you need to invest a certain amount of your valuable time into building and maintaining your development environment, your workbench so to speak. And this time gets taken away from the time you have to do actually productive things, i.e. write code. 
+Imagine you just got a new (employer provided) laptop. First thing you do is slab your favorite OS on it and then the fun begins. You have to install your favorite editor or IDE, compilers, libraries, debug tools and what have you. Fast forward a few hours and you're slowly getting to a point where your new tool becomes usable. Couple of weeks later something in your tool-chain breaks and you have to spend time to fix it. Updates need to be made, dependencies to be considered and managed and so on. Bottom line is, you need to invest a certain amount of your valuable time into building and maintaining your development environment, your workbench so to speak. And this time gets taken away from the time you have to do actually productive things, i.e. write code. 
```

Comment:
Don't understand why this distinction is needed here.

Reply:
Because engineers (myself included) like to fiddle with their own stuff. But for work are likely to be more focused on writing the code needed. Does that make sense? Am I off base?

Reply:
I think that's a nit - let's put brevity over precision I'd say.

Reply:
In general a valid point tho